src/System.Linq/src/System/Linq/Aggregate.cs

```diff
@@ -11,33 +10,77 @@ public static partial class Enumerable
     {
         public static TSource Aggregate<TSource>(this IEnumerable<TSource> source, Func<TSource, TSource, TSource> func)
         {
-            if (source == null) throw Error.ArgumentNull("source");
-            if (func == null) throw Error.ArgumentNull("func");
+            if (source == null)
+            {
+                throw Error.ArgumentNull(nameof(source));
+            }
+
+            if (func == null)
+            {
+                throw Error.ArgumentNull(nameof(func));
+            }
+
             using (IEnumerator<TSource> e = source.GetEnumerator())
             {
-                if (!e.MoveNext()) throw Error.NoElements();
```

Comment:
I'm going to leave this, at least for now.  In cases like this I don't see a problem with one-character names.